src/ImageListCatalog.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React  from 'react';
```

Comment:
Nit: A couple of these imports ended up with 2 spaces after reverting to functional components. Do a find/replace for `import React  from`